<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="styleSheet">
             <string notr="true">font: 75 italic 14pt &quot;Noto Sans&quot;;</string>
            </property>
            <property name="text">
             <string>Перенос на</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QFormLayout" name="formLayout_5">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_11"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_12">
                <property name="text">
                 <string>по X</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_6">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_13"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_14">
                <property name="text">
                 <string>по Y</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_7">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_15"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_16">
                <property name="text">
                 <string>по Z</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QPushButton" name="transfer">
                <property name="styleSheet">
                 <string notr="true">font: 81 20pt &quot;Noto Sans&quot;;</string>
                </property>
                <property name="text">
                 <string>Перенести</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLineEdit" name="lineEdit_17">
            <property name="styleSheet">
             <string notr="true">font: 75 italic 14pt &quot;Noto Sans&quot;;</string>
            </property>
            <property name="text">
             <string>Масштабирование в</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QFormLayout" name="formLayout_8">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_18"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_19">
                <property name="text">
                 <string>по X</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_9">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_20"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_21">
                <property name="text">
                 <string>по Y</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_10">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_22"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_23">
                <property name="text">
                 <string>по Z</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QPushButton" name="scaling">
                <property name="styleSheet">
                 <string notr="true">font: 81 20pt &quot;Noto Sans&quot;;</string>
                </property>
                <property name="text">
                 <string>Масштабировать</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="styleSheet">
             <string notr="true">font: 75 italic 14pt &quot;Noto Sans&quot;;</string>
            </property>
            <property name="text">
             <string>Поворот на</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_4"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_3">
                <property name="text">
                 <string>по X</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_5"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_6">
                <property name="text">
                 <string>по Y</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLineEdit" name="lineEdit_7"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_8">
                <property name="text">
                 <string>по Z</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QPushButton" name="turn">
                <property name="styleSheet">
                 <string notr="true">font: 81 20pt &quot;Noto Sans&quot;;</string>
                </property>
                <property name="text">
                 <string>Повернуть</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="loading">
              <property name="styleSheet">
               <string notr="true">font: 81 20pt &quot;Noto Sans&quot;;</string>
              </property>
              <property name="text">
               <string>Загрузить модель</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1268</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
